cli/default-reporter/src/reporterForClient/reportLifecycleScripts.ts

```diff
@@ -280,7 +280,7 @@ function cutLine (line: string, maxLength: number) {
 
 function aggregateOutput (source: Rx.Observable<LifecycleLog>) {
   return source.pipe(
-    groupBy(data => data.depPath),
+    groupBy((data) => data.depPath + '\0' + data.stage),
```

Comment:
What does \0 mean?

Reply:
`\0` is just a null character, which is a great string separator in JS since JS doesn't use null terminated strings and nothing else will really use it.

Reply:
https://github.com/blakeembrey/change-case/blob/main/packages/change-case/src/index.ts is a popular library that uses it for this purpose, for example.

Reply:
OK, sounds good. Please add a comment about it.